### ISSUE TITLE
Stretch narrow characters that are meant to be wide

### ISSUE THREAD

**Issue by GoogleCodeExporter** [mintty/mintty#123]
```
Fonts don't always match the width assumptions in wcwidth(), which MinTTY
uses to determine whether a character should be one or two cells wide. For
example, MSGothic doesn't have the double-line box drawing characters and
Windows substitutes the Lucida Console equivalents. Those are only one
rather than two cells wide though, with rather ugly consequences.

MinTTY could of course consult the font in use to determine how wide each
character is, but that would completely confuse applications. Therefore,
stretching narrow characters wide seems the best solution. (MinTTY already
deals with the opposite problem by shrinking characters that are too wide.)
```

Original issue reported on code.google.com by `andy.koppe` on 20 Jun 2009 at 6:55

**Comment by mintty** [comment]
Implemented (except for VT100 wide lines) and released in 2.4.1.

**Comment by mintty** [comment]
The tweak had to be reverted for now as it spoils wide character display (#570).

**Comment by mintty** [comment]
Released in 2.6.2.
